Cargo.toml

```diff
@@ -8,7 +8,9 @@ repository = "https://github.com/openvm-org/"
 license = "MIT OR Apache-2.0"
 
 [workspace]
-members = ["crates/stark-backend", "crates/stark-sdk"]
+members = ["crates/stark-backend", "crates/stark-sdk", "crates/cuda-common", "crates/cuda-backend", "crates/cuda-builder"]
+# Only these build by default
+default-members = ["crates/stark-backend", "crates/stark-sdk"]
```

Comment:
is it the case that you could actually include all members as default now that there is `cfg(has_cuda)`?

Reply:
I think if you are in a cuda machine, right now `cargo check` won't check the cuda crates because they aren't default